js/validation.js

```diff
@@ -0,0 +1,18 @@
+// ─── Selectors ───────────────────────────────────────────────────────────────
+const email = document.querySelector('#email');
+const form = document.querySelector('.contact-form');
+const errorMessage = document.querySelector('.errorMessage');
+
+// ─── Functions ───────────────────────────────────────────────────────────────
+function formValidation(event) {
+  event.preventDefault();
+  const emailInput = email.value;
+  if (emailInput !== emailInput.toLowerCase()) {
+    errorMessage.innerHTML = 'Please email must be in lowercase';
+    return;
+  }
+  form.submit();
+}
+
```

Comment:
- [ ] Good job :+1: so far working on this validation but please see from the screenshot below that your form is not submitting even if the entered email address is in lowercase so kindly fix it.

[Screencast from 02-04-2023 12:59:51 AM.webm](https://user-images.githubusercontent.com/92303891/216736863-d61287e3-8c71-47e7-adad-2a355b686dca.webm)

Reply:
HI @Owusu-Desmond, 
Thank you for making time to review my codes. Your feedback is much appreciated. However, Mr Ansah, hope you don't mind if I contact you for assistance when I am stuck.

Reply:
Hello @Owusu-Desmond ,
my contact form is working perfectly as expected. Please take time to check or review. Thank you.

<img width="1319" alt="Screenshot 2023-02-04 at 1 14 02 AM" src="https://user-images.githubusercontent.com/29270468/216737654-37ccb4df-dbbf-4e50-a261-cdefe6dd057a.png">

Reply:
I dont think it is working and to prevent any doubt please make a video by entering data and reply to this comment by uploading it here so I can't see it.